jvm-runtime/ftl-runtime/common/runtime/src/main/java/xyz/block/ftl/runtime/VerbHandler.java

```diff
@@ -11,14 +15,21 @@
 public class VerbHandler extends VerbServiceGrpc.VerbServiceImplBase {
 
     final VerbRegistry registry;
+    final LongCounter counter;
 
-    public VerbHandler(VerbRegistry registry) {
+    public VerbHandler(VerbRegistry registry, Meter meter) {
         this.registry = registry;
+        counter = meter.counterBuilder("ftl-jvm-runtime/verb_invocations")
```

Comment:
this appears to mirror: https://github.com/TBD54566975/ftl/blob/8453735f03dae4795a34ab0019201ddc292aa76b/backend/controller/observability/calls.go#L44

can you use the same naming scheme

Reply:
I actually wonder if we need this at all, given that we already track request numbers in the controller, it was more added just to make sure the metrics were working.

I might replace it with some user level metrics in the examples.